### DISCUSSION TITLE
Bump version of github runner with dependabot?

### DISCUSSION THREAD

**Discussion by tobiasehlert** [actions/runner#1257] [Ideas] [answered]
Hello,

Is it possible to somehow automatically create a PR base on when a new version of GitHub runner gets available?
Our runners tend to get suck and crash when they to upgrade themselves in our Kubernetes cluster and it's pretty annoying.

Someone that was a solution for this? I've created a scheduled workflow, but it feels a bit weird to do it that way.

Regards,
Tobias

**Comment by toast-gear** [comment, marked as answer]
I don't think dependabot can use a github release from another repository as a source annoyingly. `Renovate` can do this though.
